### PR TITLE
add easyconfig for GAMESS-US

### PR DESCRIPTION
**Pull request by boegel**
~~depends on https://github.com/hpcugent/easybuild-easyblocks/pull/470~~
depends on https://github.com/hpcugent/easybuild-easyblocks/pull/558

### CODE REVIEW

File: easybuild/easyconfigs/g/GAMESS-US/gamess_TMP.patch [hunk 1]
```diff
@@ -0,0 +1,36 @@
+--- gamess/compall.orig	2014-06-11 18:37:54.000000000 +0200
```
Comment (boegel) [inline easybuild/easyconfigs/g/GAMESS-US/gamess_TMP.patch]: this patch is specific to building GAMESS-US on OS X, it seems...

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c681c197d29e635e0923 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/881ef46a3d8bf502ec61 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/38785b99a5ee29797cbe for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/199dacabd7b29104e52e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b40e6833fff0d6a66110 for a full test report.

**Comment by boegel** [comment]
Hmm, it seems like the `rungms` doesn't support `OpenMPI` by default, and it's non-trivial to get it to work (cfr. http://gshamov.blogspot.be/2013/08/how-to-use-gamess-us-with-openmpi.html)

So, leaving this here for now, and going ahead with merging https://github.com/hpcugent/easybuild-easyblocks/pull/558 which does support building/testing on top of `intel` toolchain...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/bd338b902ad74e066dce for a full test report.

**Comment by boegel** [comment]
I had to skip the tests in order to make the build pass, since `rungms` doesn't provide support for using `OpenMPI`...

File: easybuild/easyconfigs/g/GAMESS-US/GAMESS-US-20130501-R1_openblas.patch [hunk 2]
```diff
@@ -0,0 +1,45 @@
+--- config.orig	2015-03-03 10:45:24.562802000 +0100
```
Comment (wpoely86) [inline easybuild/easyconfigs/g/GAMESS-US/GAMESS-US-20130501-R1_openblas.patch]: I think we should make it a habit of always adding a line of comment on top of a patch about what it does

Comment (boegel) [inline easybuild/easyconfigs/g/GAMESS-US/GAMESS-US-20130501-R1_openblas.patch]: you're right... willing to write a unit test to check that? ;)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/9ab72524c20d13f273f8 for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Going in now easyblock PR is merged @ https://github.com/hpcugent/easybuild-easyblocks/pull/558

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**